### ISSUE TITLE
The code-block should support language parameter

### ISSUE THREAD

**Issue by Shihta** [vscode-restructuredtext/vscode-restructuredtext#135]
**Environment**
- [x] Extension version: 92.0.0
- [x] Visual Studio Code version: 1.30.1
- [x] Operating System version: Mac
- [x] Python version: 2.7.15
- [ ] sphinx version: 
- [ ] A sample of conf.py or its important fragments:

**Expected behavior**

Please refer to http://www.sphinx-doc.org/en/1.5/markup/code.html#directive-code-block
The code-block should support language parameter

**Actual behavior**

![2018-12-20 11 10 54](https://user-images.githubusercontent.com/5186176/50261770-0d134080-0449-11e9-8508-c36836aedc0b.png)

Both syntax and preview window show error.

**Comment by lextm** [comment]
It is because you didn't install pygments package ("Pygments package not found"),

http://pygments.org/download/

**Comment by michaelsimonelli** [comment]
I've actually noticed something similar, that might be more direct as it what I came looking for. 
BUT
I am seeing side effects of the RST markdown creeping into my code-block.

![capture](https://user-images.githubusercontent.com/39355436/50387726-9c328980-06d1-11e9-9c73-5dfc307574aa.PNG)

Whilst I don't have the lexer loaded for this language, I would expect it to just not interfere and leave it syntax free. 

Thanks
